javascript/example-customer-account--loyalty-app--ts/extensions/customer-account-loyalty-points-block-extension/README.md

```diff
@@ -0,0 +1,3 @@
+# Customer account UI Extension
```

Comment:
Same as https://github.com/Shopify/customer-account-tutorials/pull/4/files#r1416119417

Reply:
Hey Timi, will need a bit more details here, in the link you have attached, there are two README files at the app.toml file level, one for loyalty app and one for wishlist, is this what we are looking for?